### DISCUSSION TITLE
ISCE2StaMPS Processing Error in Matlab

### DISCUSSION THREAD

**Discussion by hariiirs** [isce-framework/isce2#357] [Q&A]
I am processing Sentinel-1 images for time-series measurement using stackSentinel.py. stackSentinel.py processing is successfully completed then creation of single master stack processing (make_single_reference_stack_isce) is also completed. After this I moved to StaMPS processing (after all settings: unzip, cd StaMPS/src, make, make install and source) via mt_prep_isce 0.4 2 3 50 200, which was also successuly completed. Now while I am moving in matlab for StaMPS processing then started getting errors. These errors were tried to resolve a lot. What we exerienced is, there are two versions of StaMPS i.e. StaMPS-4.1-beta and StaMPS-master. StaMPS-4.1-beta version is not having load_isce.m file under StaMPS-4.1-beta/matlab folder, therefore we got load_isce error in matlab procesisng 

![MATLAB_Processing with StaMPS-4 1-beta](https://user-images.githubusercontent.com/42439461/136327248-5d3aeedc-5bc4-4d91-8987-ddc540613a71.png)

![StaMPS-4 1-beta_matlab_folder](https://user-images.githubusercontent.com/42439461/136327283-e72b752a-6ad0-4d95-b92a-aecdedc67f5a.png)

Then we saw, another version of StaMPS i.e. StaMPS-master where load_isce.m file is available under StaMPS-master/matlab folder and then tried the same processing in matlab. This time we got another error as shown in this screenshot-

![MATLAB_Processing with StaMPS-master](https://user-images.githubusercontent.com/42439461/136327571-0723b53e-c66c-474d-b0a8-0d87c74a3305.png)

![StaMPS-master_matlab_folder](https://user-images.githubusercontent.com/42439461/136327601-b6af3281-ff5e-4e83-a3c6-71f1eb62a41d.png)

If anybody is having any idea about to resolve this issue, kindly revert back.

Many thanks
Hari Shankar

**Comment by bjmarfito** [comment]
@hariiirs, check my post on this link: https://github.com/dbekaert/StaMPS/issues/27#issuecomment-876281891
